rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_reader :book, :person
+  attr_accessor :date
```

Comment:
- **Optional**: I would suggest you declare the variables this way: `attr_accessor: book, :person`  because person and book are both   `reader and writer`
- Here is the [link ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor%20#%20*Check%20that%20the%20instance%20variables%20have%20setters%20and%20getters)to learn more about this shortcut.